docs/pelicanconf.py

```diff
@@ -39,6 +39,12 @@
 MAIN_MENU = True
 HOME_HIDE_TAGS = True
 
+# Valid values for LINKS_IN_NEW_TAB:
```

Comment:
This comment should be in Wiki, not here, as well, this file should not be changed.

Reply:
Got it, I will revert this and add it to the wiki after the PR is merged.